design/API/NT-API.yml

```
@@ -2530,7 +2992,7 @@ components:
         content:
           type: array
           items:
-              $ref: "#/components/schemas/TaskSubmissionContentEntry"
+              $ref: '#/components/schemas/TaskSubmissionContentEntry'
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 14

(indentation)